index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+<!-- Disable zooming: -->
+<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+
+<head>
+    <title>eframe template</title>
+    <!-- the files we need to copy into the dist folder to make them available for web server -->
+    <link data-trunk rel="copy-file" href="docs/favicon.ico"/>
```

Comment:
can replace all of these with a single 
```html
<link data-trunk rel="copy-dir" href="docs">
```
and use `href="docs/favicon.ico"` as the paths. but idk if some of these files *need* to be at root folder, so i will ignore this for now.